src/scripts/main.js

```diff
@@ -1 +1,9 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
```

Comment:
Consider using 'DOMContentLoaded' event instead of 'hashchange' to handle the menu state on the initial page load as well.